mono/mini/mini-runtime.h

```diff
@@ -423,6 +423,7 @@ MONO_API void        mono_parse_env_options         (int *ref_argc, char **ref_a
 MONO_API char       *mono_parse_options_from        (const char *options, int *ref_argc, char **ref_argv []);
 MONO_API int         mono_regression_test_step      (int verbose_level, const char *image, const char *method_name);
 
+void                   print_jit_stats               (void);
```

Comment:
1. non-static functions should have a `mono_` prefix
2. doesn't seem like it's related to this PR.  This is anticipating upcoming work?

Reply:
Yes, Ryan requested that we make print_jit_stats global so we can trigger a stat dump from within the JIT or interpreter on a method call. This is anticipating the next step

Reply:
The idea is to get some kind of JIT stats printing function globally accessible for them to work with, even if it's not used yet.

Agreed that it needs to match the naming for everything else.

Whether or not we need to lock inside of this is dependent on if there's somewhere in the interpreter that is already taking the domain lock and would be convenient to put this inside of. Did you find somewhere? Or will we need to be locking ourselves? 

If we're not locking, it needs to be *extremely clear* in a comment that this expects you to already hold the domain lock. Search the codebase for other examples of this.

If we are locking, you'll need to pass in the domain and take the lock - again, examples abound elsewhere in the code. I can point you to specific examples of both of these if you can't find them.